azkaban-web-server/src/main/java/azkaban/imagemgmt/services/ImageTypeServiceImpl.java

```diff
@@ -87,7 +88,8 @@ public int createImageType(final ImageTypeDTO imageType) throws ImageMgmtExcepti
   private boolean validateOwnership(final ImageTypeDTO imageType)
       throws ImageMgmtValidationException {
     // Check if ownership record exists
-    if (imageType.getOwnerships() == null) {
+    if (CollectionUtils.isEmpty(imageType.getOwnerships())
```

Comment:
Can image type owners add owners one by one? Won't this validation stop them from doing that?

Reply:
This validation is triggered during image type creation which is one time activity. During image type creation at least two owners needs to be specified. Also we don't have separate API for adding/removing owners to start with and this will be added sometime in future. Hence, it will be good to have at least two owners to for image types. Moreover, mandate is at least one of them must be ADMIN.

Reply:
Agree with you. Thank you!